aip_xx1_launch/launch/lidar.launch.xml

```diff
@@ -23,6 +24,7 @@
         <arg name="launch_driver" value="$(var launch_driver)" />
         <arg name="vehicle_param_file" value="$(var vehicle_param_file)" />
         <arg name="vehicle_mirror_param_file" value="$(var vehicle_mirror_param_file)" />
+        <arg name="vehicle_crop_margin" value="$(var vehicle_crop_margin)" />
```

Comment:
The parameters will be reflected without this line.
Should I write it?
@tkimura4 @kenji-miyake

Reply:
If we can assume all args will be the same name, you can remove it.
But I think writing it is better since this file is a bit complicated, we shouldn't increase implicit dependencies.